src/Spec.ts

```diff
@@ -503,6 +505,94 @@ export default class Spec {
     }
   }
 
+  private async buildMultipage() {
+    const doc = this.doc.cloneNode(true) as Document;
+
+    const links = new Map<string, string>();
+    const linksForJson: { [key: string]: string[] } = {};
+    const registerLinks = (page: string, node: Element) => {
+      const all = [];
+      if (node.id) {
+        links.set(`#${node.id}`, `${page}#${node.id}`);
+        all.push(node.id);
+      }
+      Array.from(node.querySelectorAll('[id]')).forEach(n => {
+        links.set(`#${n.id}`, `${page}#${n.id}`);
+        if (!n.id.startsWith('_ref')) {
+          all.push(n.id);
+        }
+      });
+      linksForJson[page] = all;
+    };
+
+    const pages = new Map<string, Element>();
+    pages.set('index', this.doc.createElement('div'));
+    Array.from(doc.querySelector('#spec-container')!.children).forEach(c => {
+      if (!c.id || !/^(\d+|\w) /.test(c.querySelector('h1')?.textContent!)) {
```

Comment:
is there a better way to tell apart whether the section is standalone (`1 Scope`, `A Grammar Summary`) or should be part of the landing page?

Reply:
I'm not sure what distinction you're drawing. Do you just mean "how do I identify the introduction"? It's the bit wrapped in an `emu-intro` tag instead of an `emu-clause` tag.

Reply:
it's more like, the intro and everything before the intro (the title, logo, metadata block, etc). is it safe to just do `if (has not seen 'emu-intro')`?

Reply:
I believe that holds currently, though you should check it and issue a warning ([like this](https://github.com/tc39/ecmarkup/blob/40d56a311165bb974e75ab50b8803115ee560700/src/Spec.ts#L530-L537)) if it's violated, rather than just assuming it.